pages/partials/navbar.ejs

```diff
@@ -1,126 +1,114 @@
 <nav class="navbar navbar-dark bg-dark navbar-expand-md mb-4">
-  <a class="navbar-brand" href="<%= homeUrl %>">
-    <span class="navbar-brand-label">PrairieLearn</span>
-    <span class="navbar-brand-hover-label text-secondary">Go home <i class="fa fa-angle-right" aria-hidden="true"></i></span>
-  </a>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse" aria-expanded="false" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
-  </button>
-  <div class="collapse navbar-collapse">
-    <ul class="nav navbar-nav mr-auto" id="main-nav">
-        <% if (typeof navbarType == 'undefined' || navbarType == 'plain') { %>
-        <%- include('navbarPlain'); %>
-        <% } else if (navbarType == 'student') { %>
-        <%- include('navbarStudent'); %>
-        <% } else if (navbarType == 'instructor') { %>
-        <%- include('navbarInstructor'); %>
-        <% } else { %>
-        <% throw new Error('Unknown navbarType: ' + navbarType); %>
-        <% } %>
-    </ul>
-
-    <!-------------------------------------------------------------------------------->
-    <!-------------------------------------------------------------------------------->
-    <!-------------------------------------------------------------------------------->
-    <!-- User and logout ------------------------------------------------------------->
-
-<%
-if (!locals.effective_user_has_no_access_to_course_instance) {
-  var displayedName = '';
-  if (typeof authz_data !== 'undefined' && authz_data.authn_has_instructor_view) {
-      if (authz_data.user.name) {
-          displayedName = authz_data.user.name;
-      } else {
-          displayedName = authz_data.user.uid;
-      }
-      displayedName += ' (' + authz_data.role;
-      if (authz_data.mode != 'Public') {
-          displayedName += ', ' + authz_data.mode;
-      }
-      displayedName += ')';
-  } else {
-      if (typeof authn_user == 'undefined') {
-          displayedName = '(No user)';
-      } else if (is_administrator) {
-          displayedName = authn_user.name + ' (Administrator)';
-      } else {
-          displayedName = authn_user.name;
-      }
-  }
-}
-%>
-
-<ul class="nav navbar-nav" id="username-nav">
-
-    <% if (!locals.effective_user_has_no_access_to_course_instance) { %>
-    <li class="nav-item dropdown mb-2 mb-md-0 mr-2 <% if (navPage == "effective") { %>active<% } %>">
-        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            <%= displayedName %>
-            <% if (typeof news_item_notification_count != 'undefined' && news_item_notification_count > 0) { %>
-              <span class="badge badge-pill badge-primary news-item-count"><%= news_item_notification_count %></span>
+    <!-- Home link -->
```

Comment:
This commit is entirely just balancing some messy indentation and comments in the navbar files.